nixos/modules/services/hardware/kmonad.nix

```diff
@@ -41,6 +41,19 @@ let
           '';
         };
 
+        enableHardening = lib.mkOption {
+          type = lib.types.bool;
+          default = true;
```

Comment:
Do you think enabling hardening is a good idea even if hardening may make some shell commands fail?

Reply:
I think it makes sense to have these options by default. For people that want to use `cmd-button`, they can disable this and add the options that are compatible with their macros themselves. 

I will test it on my system and come back to you.